### DISCUSSION TITLE
HTTP and HTTPS on difrent port

### DISCUSSION THREAD

**Discussion by missosss123** [NginxProxyManager/nginx-proxy-manager#2653] [General]
Good day.

I would like to ask if it is possible on jc21/nginx-proxy-manager to change port 80 to 82 and 443 to 444 like this:

version: '3'

services:

app:

image: 'jc21/nginx-proxy-manager:latest'

ports:

- '82:80'

- '83:81'

- '444:443'

environment:

DB_MYSQL_HOST: "db"

DB_MYSQL_PORT: 3306

DB_MYSQL_USER: "usr"

DB_MYSQL_PASSWORD: "passwd"

DB_MYSQL_NAME: "npm"

volumes:

- ./data:/data

- ./letsencrypt:/etc/letsencrypt

db:

image: 'jc21/mariadb-aria:latest'

environment:

MYSQL_ROOT_PASSWORD: 'passwd'

MYSQL_DATABASE: 'npm'

MYSQL_USER: 'usr'

MYSQL_PASSWORD: 'passwd'

volumes:

- ./data/mysql:/var/lib/mysql

**Comment by jlmanohar** [comment]
It should be possible from what i know (Docker doesn't block that kind of config), but the question is, what advantage does it offer you ? 
As far as i know all browsers default to port 80 for http and 443 for https. So it would be a pain to access any site or app you would be proxying.

**Reply by missosss123** [reply]
I curently have 2 servers and i wanna know if i could somehow ide rporxy for 2 servers atvsame tíme because if i occupied port 80 and 443 with server 1 i cannot access second one...

**Reply by missosss123** [reply]
Aby sugestion because the servers or better onene must stay online even if i vant just access second one

**Reply by jlmanohar** [reply]
What is your setup ? Do you have both the servers on your local network, both exposed on public ?

Also if you want to access services on service 2, you can do it with just one instance (i.e. you dont need npm on server1 and server2)

**Reply by jlmanohar** [reply]
Example

server 1 - only on local (port 80, 443 used by some other service)
server 2 - on local, but has port forwarding enable in router (available on internet)

install npm on server2 
Whatever domains you want to route to an app on server1 configure it like you would regularly, except give ip address on server 1 as the route destination

A catch is that server1 and 2 should be abl;e to route to each other and not blocked physically or through firewall

**Reply by missosss123** [reply]
Yes i use it this way bit if server 1 is down that mean the work ok server 2 is unusable
